app/models/buffer_update.rb

```diff
@@ -5,9 +5,11 @@ class BufferUpdate < ApplicationRecord
   validate :validate_body_text_recent_uniqueness, :validate_suggestion_limit
   validates :status, inclusion: { in: %w[pending sent_direct confirmed dismissed] }
 
-  def self.buff!(
-    article_id, text, buffer_profile_id_code, social_service_name = "twitter", tag_id = nil, admin_id = nil
-  )
+  def self.buff!(article_id, text, buffer_profile_id_code, **options)
+    social_service_name = options.fetch(:social_service_name, "twitter")
+    tag_id = options.fetch(:tag_id, nil)
+    admin_id = options.fetch(:admin_id, nil)
```

Comment:
I don't think you need to use fetch here. `kwargs[:tag_id]` is perfectly fine. As it's going to be nil anyway if the value is not given